test/fixtures/krb5kdc-fixture/src/main/resources/provision/addprinc.sh

```diff
@@ -19,6 +19,9 @@
 
 set -e
 
+krb5kdc
+kadmind
```

Comment:
Should these be in the hdfs.sh instead? Do these commands have any issues with being executed multiple times when adding multiple principals for a fixture?

Reply:
To clarify "in hdfs.sh": I mean "in provision/hdfs.sh and provision/peppa.sh"

Reply:
I tried it but the containers don't come online. since the tests work as-si I didn't look further at it, we can perhaps address this lather on.